lib/api.js

```diff
@@ -20,6 +20,7 @@ function PlexAPI(options, deprecatedPort) {
     this.hostname = hostname;
     this.port = deprecatedPort || opts.port || PLEX_SERVER_PORT;
     this.https = opts.https;
+    this.timeout = opts.timeout || 60000;
```

Comment:
Makes sense! Not 100% sure if we should remove the other side of the `||` or specifically set it to `null/undefined`?